### ISSUE TITLE
Package view tabs doesn't have role="button" 

### ISSUE THREAD

**Issue by jonasfj** [dart-lang/pub-dartlang-dart#1975]
Package view tabs:

![7kdtyaamtgu](https://user-images.githubusercontent.com/149732/51473798-a356c100-1d7d-11e9-84c5-53999ac4493c.png)

Does not have `role="button" ` this improves accessibility:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

(note, I only noticed this because I could select them with the vimium extension for chrome).
